automation/test.sh

```diff
@@ -170,6 +170,16 @@ fi
 
 kubectl() { cluster-up/kubectl.sh "$@"; }
 
+collect_debug_logs() {
+    local containers
+
+    containers=( $(docker ps --format '{{ .Names }}') )
```

Comment:
i believe you need -a
since the wanted container is probably dead
i will try to simulate it locally as well, injecting an error

Reply:
Added the `-a` flag.